api/src/main/java/org/apache/cloudstack/api/command/admin/acl/UpdateRoleCmd.java

```diff
@@ -67,13 +63,6 @@ public String getRoleName() {
         return roleName;
     }
 
-    public RoleType getRoleType() {
-        if (!Strings.isNullOrEmpty(roleType)) {
-            return RoleType.fromString(roleType);
-        }
-        return null;
-    }
-
     public String getRoleDescription() {
```

Comment:
Since roleType and roleDescription have been moved to RoleCmd class, can we not remove getRoleDescription() from this class?

Reply:
> Since roleType and roleDescription have been moved to RoleCmd class, can we not remove getRoleDescription() from this class?

thanks @Pearl1594, missed this one. updated.